src/DatetimePopup.vue

```diff
@@ -3,6 +3,8 @@
     <div class="vdatetime-popup__header">
       <div class="vdatetime-popup__year" @click="showYear">{{ year }}</div>
       <div class="vdatetime-popup__date">{{ dateFormatted }}</div>
+      <div class="vdatetime-popup__year" @click="showYear" v-if="type !== 'time'">{{ year }}</div>
+      <div class="vdatetime-popup__date" v-if="type !== 'time'">{{ dateFormatted }}</div>
```

Comment:
Something is wrong here. Headers are duplicated.
What is your idea for the header? Show an empty header?

Reply:
I think its by mistake, not removed. Should be [just](https://github.com/a2exfr/vue-datetime/blob/62460e5a69a3a30c30c8fabb19f616a7eedba0e7/src/DatetimePopup.vue#L3-L5)
The idea was  to show small colored  header line. With [this](https://github.com/a2exfr/vue-datetime/blob/v1.x/src/DatetimePopup.vue#L264).

Reply:
actually i was able to quickly get that fixed.